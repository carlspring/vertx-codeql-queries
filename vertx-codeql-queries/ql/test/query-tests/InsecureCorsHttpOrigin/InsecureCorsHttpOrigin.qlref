vertx-codeql-queries/InsecureCorsHttpOrigin.ql
