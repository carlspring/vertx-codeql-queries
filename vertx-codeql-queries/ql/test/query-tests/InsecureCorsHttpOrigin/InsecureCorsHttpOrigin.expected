| createHttpServer(...) | Insecure HTTP server which allows unencrypted HTTP connections |
